### PR TITLE
Add glossary theming

### PR DESCRIPTION
**Pull request by shannynalayna**
Signed-off-by: Shannyn Telander <telander@bitwise.io>

### CODE REVIEW

File: css/custom.css [hunk 1]
```diff
@@ -461,3 +461,19 @@ h3 {
...
 }
+
+.glossary-header {
+    color: var(--splinter-fuchsia);
+    line-height: 0px;
+    font-size: 20px;
+}
+
+.glossary-definition {
+    color: var(--text-color);
```
Comment (ltseeley) [inline css/custom.css:472]: This line isn't necessary

File: docs/0.4/glossary/glossary.md [hunk 2]
```diff
@@ -6,79 +6,143 @@
...
-## Biome
+<h3 class="glossary-header" id="biome">
+Biome
+</h3>
 
-Biome is a module in libsplinter containing several submodules that provide
-support for user management, user credential management, and private key
-management.
+<p class="glossary-definition">
+<b class="glossary-entry">Biome</b> is a module in libsplinter containing
```
Comment (ltseeley) [inline docs/0.4/glossary/glossary.md:14]: I'm not sure making this `glossary-entry` fuchsia or bold is necessary; IMO it's visually distracting, and the word is already in big fuchsia letters in the H3.

Comment (shannynalayna) [reply docs/0.4/glossary/glossary.md:14]: I was basing this off of glossary examples that we had talked about, mainly 
https://pulsar.apache.org/docs/en/reference-terminology/, but if you think that it's distracting, I am not tied to this design.

**Comment by vaporos** [comment]
header-grey should be fine for now

**Pull request merged**